src/Effect/Aff.js

```diff
@@ -56,6 +56,12 @@ var Aff = function () {
   var FIBER     = "Fiber";     // Actual fiber reference
   var THUNK     = "Thunk";     // Primed effect, ready to invoke
 
+  // Error used for early cancelation on Alt branches.
+  // This is initialized here (rather than in the Fiber constructor) because
+  // otherwise, in V8, this Error object indefinitely hangs on to memory that
+  // otherwise would be garbage collected.
+  var early = new Error("[ParAff] Early exit");
```

Comment:
Good find. Any thoughts as to why this happens? Just curious.

Reply:
I would guess it has something to do with stack frames and traces or whatever.

Reply:
`<symbol> in Error` is what is hanging onto the memory, whatever that is, and I do not know why `Error` is being referenced from the root. I figure it is some special debugging sauce in V8 being quirky.